src/main/java/org/cascadebot/cascadebot/commands/moderation/ModlogDisableSubCommand.java

```diff
@@ -0,0 +1,143 @@
+package org.cascadebot.cascadebot.commands.moderation;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.entities.TextChannel;
+import org.apache.commons.lang3.EnumUtils;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.commandmeta.SubCommand;
+import org.cascadebot.cascadebot.data.language.Language;
+import org.cascadebot.cascadebot.messaging.MessageType;
+import org.cascadebot.cascadebot.moderation.ModlogEvent;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+import org.cascadebot.cascadebot.utils.DiscordUtils;
+import org.cascadebot.cascadebot.utils.pagination.Page;
+import org.cascadebot.cascadebot.utils.pagination.PageObjects;
+import org.cascadebot.cascadebot.utils.pagination.PageUtils;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ModlogDisableSubCommand extends SubCommand {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        if (context.getArgs().length < 2) {
+            context.getUiMessaging().replyUsage();
+            return;
+        }
+        TextChannel textChannel = DiscordUtils.getTextChannel(context.getGuild(), context.getArg(0));
+        if (textChannel == null) {
+            context.reply("Invalid channel");
+            // Invalid channel
+            return;
+        }
+        List<String> events = new ArrayList<>(Arrays.asList(context.getArgs()));
+        events.remove(0);
+        List<ModlogEvent> modlogEvents = new ArrayList<>();
+        List<String> failed = new ArrayList<>();
+        for (String event : events) {
+            if (EnumUtils.isValidEnum(ModlogEvent.class, event.toUpperCase())) {
+                modlogEvents.add(ModlogEvent.valueOf(event.toUpperCase()));  // TODO reverse language to get event
```

Comment:
I might make language util that enables this as it's common. I'm thinking just have a enum cache like: 
```java
Map<Locale, Map<Enum, String>>
```